gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -43,17 +43,14 @@ class VeloxTestSettings extends BackendTestSettings {
       "rollup overlapping columns", // wait velox to fix
       "cube overlapping columns", // wait velox to fix
       // incorrect result, distinct NaN case
-      "SPARK-32038: NormalizeFloatingNumbers should work on distinct aggregate",
-      // decimal failed ut
-      "SPARK-36926: decimal average mistakenly overflow"
+      "SPARK-32038: NormalizeFloatingNumbers should work on distinct aggregate"
     )
 
   enableSuite[GlutenCastSuite]
     .exclude(
       "Process Infinity, -Infinity, NaN in case insensitive manner" // +inf not supported in folly.
     )
     // decimal failed ut
-    .exclude("Fast fail for cast string type to decimal type")
```

Comment:
Can we rebase to main and enable all the left varchar-to-decimal unit tests?

Reply:
Fixed all the comments